### DISCUSSION TITLE
Dataset Split Size

### DISCUSSION THREAD

**Discussion by kinredon** [p-lambda/wilds#65] [Q&A] [answered]
I noted that the dataset has been updated, e.g. iwildcam. Where can I find the latest information about the dataset split size?

**Comment by kohpangwei** [comment, marked as answer]
The updates are described in the release notes: https://github.com/p-lambda/wilds/releases
and also reflected in the latest arxiv paper: https://arxiv.org/abs/2012.07421

Hope that helps!
